ledger/internal/applications.go

```diff
@@ -201,11 +201,11 @@ func (cs *roundCowState) kvGet(key string) (string, bool, error) {
 	if !ok {
 		return cs.lookupParent.kvGet(key)
 	}
-	if value == nil {
+	// If value is nil, it's a marker for a local deletion
```

Comment:
I'm surprised that the rest of this function (past L203) is not covered by any unit tests according to **Codecov**. Is that a concern?

Reply:
Looking into it. I'm think we just don't count coverage properly across packages (#3185).  This should be tested by the `box_del` in `boxtxn_test.go` but that is in `ledger` and this is the ledger `internal` package's code.  I just learned about this limitation though, so I'll investigate to ensure that's really what's going on.

Reply:
I've confirmed this is being tested by putting a panic() in there, running the tests, and seeing the panic.  This is indeed because it's a cross package test.

Reply:
Thanks for this follow up. I'll put up a question on dev to see if there's a way to configure Codecov to handle cross-package testing.

Reply:
There is, but it's apparently too slow.  Read the issue I linked above.